src/components/styles/BookForm.module.css

```diff
@@ -1,3 +1,58 @@
 * {
   margin: 0;
 }
+
```

Comment:
- [ ] On your `styles/` folder, you are repeating several CSS rules on the same files.  This happens on all of them `Book.module.css`, `BookForm.module.css`, `BookStore.module.css`, and `Navbar.module.css`. 

Please, don't repeat your CSS rules on each file, to keep your code is DRY.

## Explanation

This is an example of the issue on one of the 4 files:

![image](https://user-images.githubusercontent.com/57421823/155658820-5f946612-d4dc-4727-bcab-25b71d6513e9.png)

## Hints

There are 2 solutions for this issue according to this [support article](https://speckyboy.com/good-bad-css-practices/):

1. Use a class instead:

![image](https://user-images.githubusercontent.com/57421823/155659112-1b9f1b58-8efb-4de5-b1e8-d7d2cc6cd9b3.png)

2. Combine the elements:

![image](https://user-images.githubusercontent.com/57421823/155659144-6d0ba5ab-ac31-43bd-af7e-43855dfde3b0.png)


If you implement this, you will achieve the best CSS practices like a pro 💪

Reply:
thanks a lot for your review. The best one so far in my microverse journey. Great job

I will make the changes as soon as possible

Reply:
Thanks a lot for your effort, and for your kind words. 🙏

Reply:
Hi @Meltrust I made the changes but now a have a weird error that it wasn't in .. 😖